DigitalLearningSolutions.Web/Styles/layout.scss

```diff
@@ -338,3 +338,8 @@ nav, .nhsuk-header__navigation, #header-navigation {
 .nhsuk-width-container {
   max-width: 1144px !important;
 }
+@media only screen and (max-width: 767px) {

```

Comment:
This is going to hit all `.nhsuk-grid-column-one-third` elements throughout the application. Is that the desired effect? Have we tested other examples in the application? Would it be safer to apply a `.nhsuk-u-margin-top...` class to the elements effected by the bug?

Reply:
This is fixed as per the comment in jira